src/components/Profile/ProfileComponent.jsx

```diff
@@ -158,8 +157,7 @@ const ProfileComponent = ({ contactProfile, webId }) => {
                 aria-label="Copy Invite Link"
                 edge="end"
                 onClick={() => {
-                  navigator.clipboard.writeText(signupLink);
-                  addNotification('success', 'Invite link copied to clipboard');
+                  saveToClipboard(signupLink, 'Invite link copied to clipboard', addNotification);
```

Comment:
Did a copy and paste of that new link and it doesn't seem to be matching the URL next to the button.

Looking that the Link above at Line 153, the link should really be `${signupLink}?webId=${encodeURIComponent(session.info.webId)}` I think. The `signupLink` might just be the initial portion of that link and doesn't actually return the full invite link.